k-distribution/src/main/scripts/lib/pyk/pyk/kastManip.py

```diff
@@ -641,7 +640,8 @@ def buildRule(ruleId, initConstrainedTerm, finalConstrainedTerm, claim=False, pr
     ruleBody = pushDownRewrites(KRewrite(initConfig, finalConfig))
     ruleRequires = simplifyBool(unsafeMlPredToBool(initConstraint))
     ruleEnsures = simplifyBool(unsafeMlPredToBool(finalConstraint))
-    ruleAtt = None if claim or priority is None else KAtt(atts={'priority': str(priority)})
+    attDict = {} if claim or priority is None else {'priority': str(priority)}
+    ruleAtt = KAtt(atts=attDict)
```

Comment:
This was where we were getting `None` values for `rule.att`. I've added a test to called `BuildRuleTest` which exercises this.